backend/de.metas.adempiere.adempiere/base/src/main/java-gen/org/compiere/model/I_C_Project.java

```diff
@@ -1098,6 +1097,27 @@ public interface I_C_Project
 
 	String COLUMNNAME_SalesRep_ID = "SalesRep_ID";
 
+	/**
+	 * Set Specialist Consultant.
+	 *
+	 * <br>Type: String
+	 * <br>Mandatory: false
+	 * <br>Virtual Column: false
+	 */
+	void setSpecialist_Consultant (@Nullable java.lang.String Specialist_Consultant);
```

Comment:
so the Specialist_Consultant is, in the end, metasfresh generic?

Reply:
@MHNV cc @metas-rc  also why is string?!